src/DigitalSubscriberAppBanner.tsx

```diff
@@ -283,19 +286,45 @@ export const storeIcon = css`
     }
 `;
 
+const withTryCatchBehaviour = (fn: () => void): void => {
+    try {
+        fn();
+    } catch (e) {}
```

Comment:
It's always a huge red flag to me to see errors being silently swallowed like this! Should we not be doing something with these errors? Printing to the console in a pinch, and ideally sending to Sentry. I feel like this is something we will regret in the future when it becomes difficult to debug inconsistent problems and strange behaviour.

I'd also observe that this says `withTryCatchBehaviour`, but by not providing the ability to recover this isn't really giving us try/catch behaviour at all. This might be more accurately named as something like "swallowErrors"?

Reply:
Good shout. I agree that ideally we want to be passing through an error handler or similar from the platforms (I commented on this in the commit message, but that wasn't very visible - apologies!) I think rather than do too much in a single PR console logging the error would be a good compromise :+1:

Reply:
I've started logging any caught errors (and changed the name of the wrapper function, which I've kept, to be clearer): d73ae90197a3da0f45707ba4be279abf2b895707